civihr_employee_portal/features/civihr_employee_portal_features/civihr_employee_portal_features.pages_default.inc

```diff
@@ -185,29 +185,6 @@ function civihr_employee_portal_features_default_page_manager_pages() {
   $display->content['new-48897dc2-d49e-4e72-b86a-7ec70b28da04'] = $pane;
   $display->panels['header'][0] = 'new-48897dc2-d49e-4e72-b86a-7ec70b28da04';
   $pane = new stdClass();
-  $pane->pid = 'new-80f12630-09c3-4ce3-8d65-f766db9a08a9';
```

Comment:
Every time you change something in a feature, you need to add an upgrader to revert it. Please add one for this change. Also, please confirm that by reverting the feature this pane will be removed from the site. If it's not, then you'll have to add an upgrader to do that.

Reply:
I have checked and the pane no longer exist.

Reply:
@ajesamson I don't see an upgrader to revert the feature. Are you saying that the pane was gone without having to revert the feature?